### PR TITLE
Added Article - Can a cancer patient live a normal life?

### LINKED ISSUE #144: 📝[Docs]:  Can a cancer patient live a normal life?
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

You have to prepare an analysis for the following topics in a markdown file.

### Can a cancer patient live a normal life?

(i) How different is their life from the daily life of an ordinary person?
(ii) How much extra care and attention they needed.
(iii) Their trauma, anxiety, depression, and their solutions, How they can keep their mind fresh.
(iv) Is cancer genetic?
(v) Is cancer contagious?
 

### Suggested Change

I would like to make it more well organise. If needed please assign this issue to me.

### Rationale

_No response_

### Urgency

Medium

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

@ananyag309 Please let me know whether you are available for the issue.

### PR DESCRIPTION
**Pull request by ananyag309**
#### PR Description
[Docs]:  Can a cancer patient live a normal life? (Issue #144 )

#### Type of Change
- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [yes ] Documentation update

#### Select your work-area
- [ ] Frontend
- [ ] Backend
- [yes ] Documentation
- [ ] Others

#### Fixes (mention the issue number which this fixes)
#closes #144 

#### Checklist
- [yes ] I have updated my branch and synced it with the project's 'develop' branch before making this PR.
- [yes ] I have optimized the file changes.
- [yes ] I have added a snapshot of my work example.
- [yes ] I have made a PR to the project's develop branch.

#### Undertaking
- [y ] My code follows the style guidelines of this project.
- [ y] I have performed a self-review of my code.
- [ y] I have commented my code, particularly in hard-to-understand areas.
- [ y] I have made corresponding changes to the documentation.
- [y ] I have checked for plagiarism and assure its authenticity.
- [y ] I have read and followed the code of conduct for this repository. I understand that violation of this undertaking may have legal consequences.

- [y ] I Agree

### CODE REVIEW

**Comment by ananyag309** [comment]
@SB2318 ma'am please have a check and let me know if any changes are to be made

File: DataSources/Article.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+4. [Their Trauma, Anxiety, Depression, and Their Solutions, How They Can Keep Their Mind Fresh](#3-their-trauma-anxiety-depression-and-their-solutions-how-they-can-keep-their-mind-fresh)
+5. [Is Cancer Genetic?](#4-is-cancer-genetic)
+6. [Is Cancer Contagious?](#5-is-cancer-contagious)
+7. [Case Studies](#case-studies)
+8. [Conclusion](#conclusion)
+10. [References](#references)
+
+---
+
+## Abstract
```
Comment (SB2318) [inline DataSources/Article.md:16]: What is Cancer?
Cancer History
Origin
How cancer is formed?  (Include some picture)

https://academic.oup.com/book/27252/chapter-abstract/196863994?redirectedFrom=fulltext

Comment (ananyag309) [reply DataSources/Article.md:16]: ok i will make the changes soon

File: DataSources/Article.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+Cancer profoundly impacts the lives of individuals diagnosed with the disease, presenting challenges across physical, emotional, and social dimensions. This paper explores the extent to which cancer patients can maintain normalcy in their lives, examining daily adjustments, care requirements, psychological impacts, genetic predispositions, and misconceptions about the disease.
+
+---
+
+## 1. How Different is Their Life from the Daily Life of an Ordinary Person?
+
+### Daily Routine Adjustments
```
Comment (SB2318) [inline DataSources/Article.md:23]: Separate it from the daily life of normal people. Actually, individuals living with cancer often face additional challenges in their daily lives. However, it is important to remember that they are not alone in their journey. Through your articles, your aim to **provide hope and support to cancer patients**, letting them know that they are not separate creatures but rather individuals who can relate to and understand their experiences.  This feeling is missing.

Comment (ananyag309) [reply DataSources/Article.md:23]: ok i got it!! i will reframe it all again with the new tone of context

**Comment by SB2318** [comment]
👍

**Comment by ananyag309** [comment]
@SB2318 ma'am changed a few things, this topic is so vast and there is more things we can include. please once go through it and let me know if it's enough or there are still things we can add. please guide.

**Comment by SB2318** [comment]
> @SB2318 ma'am changed a few things, this topic is so vast and there is more things we can include. please once go through it and let me know if it's enough or there are still things we can add. please guide.

No, you don't need to cover everything.
Give a brief description of every point.

Expands only those points which I have mentioned in the review.
Share your own view, don't blindly follow AI.

**Review by SB2318 — APPROVED** [review]
Very Good ! I am speechless. One of the best contribution you have made. 💕

Thanks!

**Comment by SB2318** [comment]
Thanks for your effort! ❤️

**Pull request merged**